docs/source/install.rst

```diff
@@ -72,8 +70,8 @@ and :ref:`next steps <next-steps>` such as running the Jupyter Notebook.
 
 .. _existing-python-new-jupyter:
 
-If you are an existing Python user but new to Jupyter
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Experienced Python user - New to Jupyter
```

Comment:
I like how you have rephrased these headings. But I would probably use colons like this "Experience Python user: new to Jupyter" on each of them.